pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -304,8 +304,8 @@ func runTPCC(
 				MaybeFlag(opts.DB != "", "db", opts.DB).
 				Flag("warehouses", opts.Warehouses).
 				MaybeFlag(!opts.DisableHistogram, "histograms", histogramsPath).
-				MaybeFlag(t.ExportOpenmetrics(), "histogram-export-format", "openmetrics").
-				MaybeFlag(t.ExportOpenmetrics(), "openmetrics-labels", clusterstats.GetOpenmetricsLabelString(t, c, labelsMap)).
+				MaybeFlag(!opts.DisableHistogram && t.ExportOpenmetrics(), "histogram-export-format", "openmetrics").
```

Comment:
Since the other openmetrics related flags also shouldn't be set when histogram is disabled.
Including the change here itself.